scripts/dev/find_byref_bool_override.py

```diff
@@ -96,7 +96,8 @@
 # Files for which to ignore missing header warning
 EXPECT_MISSING_HEADER = ['src/EnergyPlus/main.cc',
                          'src/EnergyPlus/test_ep_as_library.cc',
-                         'EnergyPlusPgm.cc']
+                         'EnergyPlusPgm.cc',
+                         'src/EnergyPlus/WindowsGuiLauncher.cc']
```

Comment:
A new source file that doesn't have a matching header, since it's just a tiny little exe.